Source/Kesmai.Server/Game/Entities/Mobiles/MobileEntity.cs

```diff
@@ -22,6 +22,25 @@ public int CalculateDamageModifier(ItemEntity item)
 			
 			if (armor != null)
 				damageModifier += armor.GetArmorBonus(item);
+			
+			/* Calculate armor bonus from held items. */
+			var leftHand = LeftHand;
+			var rightHand = RightHand;
+
+			if (leftHand is Shield shield)
```

Comment:
should shield work in right hand as well?  i could see someone doing rhammer shield and wanting the / to work for throw over the * or something

Reply:
you can just have the raxe in main hand and hsield left and you can use both * and /